<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聚师测试工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/Users/Desktop/微信图片_20220406101419.jpg</normaloff>D:/Users/Desktop/微信图片_20220406101419.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>791</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QToolButton" name="btn_addOrder">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>添加订单</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btn_addClass">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>班级添加学员</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btn_pushOrder">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>销售订单推送至聚分析</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btn_suchClass">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询班级</string>
        </property>
       </widget>
       <widget class="QToolButton" name="btn_suchUserId">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>班级一键发布</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>651</width>
          <height>451</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_4">
         <property name="font">
          <font>
           <family>Bahnschrift SemiBold</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>90</y>
            <width>211</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请左侧选择功能</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_addOrder">
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加订单</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加数量</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>100</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>事业部</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>150</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手机号</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>190</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>班级id</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>230</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>是否审核</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="edit_log">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>350</y>
            <width>651</width>
            <height>101</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_sub">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>202</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_addOrder_clear">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>250</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_phone">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>160</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="box_store">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>110</y>
            <width>141</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="box_isAudit">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>240</y>
            <width>141</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_num">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>等同于添加对公支付订单+审核</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_classId">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>200</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>添加班级学员</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>53</x>
            <y>90</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>班级id</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>150</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>目标事业部</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>210</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学员手机号</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_addClass_push">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>300</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_addClass_skuId">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>90</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_addClass_phoneNum">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>210</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="box_addClass_store">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>150</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="edit_addClass_log">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>370</y>
            <width>451</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>20</y>
            <width>251</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>为指定学员添加班级</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>311</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>销售订单推送到据分析（千帆）</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>130</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>订单id</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_push">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>200</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>推送</string>
          </property>
         </widget>
         <widget class="QLabel" name="edit_pushLog">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>270</y>
            <width>371</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_orderId">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>140</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_suchClass">
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请输入班级名称</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edit_suchClass_OrderId">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>271</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_suchClassByOrder">
          <property name="geometry">
           <rect>
            <x>460</x>
            <y>20</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_lastPage">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>410</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>上一页</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_nextPage">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>410</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>下一页</string>
          </property>
         </widget>
         <widget class="QLabel" name="text_page">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>410</y>
            <width>54</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0/0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTableWidget" name="table_class">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>651</width>
            <height>322</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>班级名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>售价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>班级id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>产品id</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="btn_clear">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>20</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>新宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>班级一键发布</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前登录平台：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>实用工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>测试地址</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>数据分析</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_logout">
    <property name="title">
     <string>退出登录</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_logout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
